src/Tribe/Service_Providers/Widgets.php

```diff
@@ -0,0 +1,87 @@
+<?php
+namespace Tribe\Service_Providers;
+
+use Tribe\Widget\Manager;
+
+/**
+ * Class Widget
+ *
+ * @since   tBD
```

Comment:
Noting this here, but the replace command is case insensitive so this is technically fine

Reply:
```suggestion
 * @since   TBD
```

Reply:
For the sake of uniformity.